pkg/agent/client.go

```diff
@@ -73,6 +73,16 @@ func (cm *connectionManager) Delete(connID int64) {
 	delete(cm.connections, connID)
 }
 
+func (cm *connectionManager) List() []*connContext {
+	cm.mu.Lock()
+	defer cm.mu.Unlock()
+	connContexts := make([]*connContext, 0, len(cm.connections))
+	for _, connCtx := range cm.connections {
```

Comment:
Might be better to return pairs of ConnectionID and ConnContexts.

Reply:
Given https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/163#discussion_r573280986 please ignore this comment.